test/spec/index.spec.js

```diff
@@ -529,4 +529,10 @@ describe('bookshelf-model-base-plus', () => {
         expect(error.message).toBe(errMsg);
       });
     });
+
+    describe('security', () => {
+      test('should treat query strings as strings not as js reserved words', async () => {
+        await User.getList({ first_name: 'constructor' }, User.columns);
```

Comment:
Where is the expect statement?

Reply:
without the fix it throws an error

Reply:
`expect(() => ...).not.toThrow()`
https://jestjs.io/docs/en/expect#tothrowerror

Reply:
ok I’ll add that